editions/tw5.com/tiddlers/system/wikitext-macros.tid

```diff
@@ -75,7 +75,10 @@ type: text/vnd.tiddlywiki
 <tr>
 	<th><<id>></th>
 	<td><$codeblock code=<<code>>/></td>
-	<td><<code>></td>
+	<td>
+
+		<<code>>
```

Comment:
Would it be possible to use `<$transclude $variable="code" $mode="block"/>` instead of the "brittle" linebreak, that is not obvious, why it is there. The next time someone touches that code it will probably been gone again.

Reply:
Thanks, @pmario. Good idea. I made the suggested change.